sbi/inference/posterior.py

```diff
@@ -93,6 +94,16 @@ def __init__(
         # Correction factor for leakage, only applicable to SNPE-family methods.
         self._leakage_density_correction_factor = None
 
+    def focus_training_on(self, x) -> "NeuralPosterior":
```

Comment:
I'm seeing no invocation of this method in the code, shouldn't it be called at some point?

Reply:
The user has to call it after inference.  See the API example in the PR description.

Reply:
Ah, I see. I missed the edited version. How about sticking to method names that unspecific to training/inference?

Reply:
For example:
```
# Single round inference with prior as proposal
posterior = infer(num_simulations=200, proposal=None)   # proposal=None is also default.
posteriors = [posterior]

# Run additional rounds with the last posterior as new proposal.
for round_ in range(1, num_rounds):
    posteriors.append(infer(num_simulations=200, proposal=posteriors[round_-1].set_default_x(x_o)))
```

Reply:
Also, we probably want: `range(1, num_rounds + 1)`

Reply:
Or shorter:
```
posteriors = []
proposal = None
for _ in range(num_rounds):
    posterior = infer(num_simulations=200, proposal=proposal)
    proposal = posterior.set_default_x(x_o)
    posteriors.append(posterior)
```

Reply:
More generally, I wonder if `infer()` should keep a `num_rounds` keyword and build such a loop internally